### ISSUE TITLE
Gluttony Wiki Page Corections

### ISSUE THREAD

**Issue by yeelp** [yeelp/Scaling-Feast#59]
GitHub is being weird right now and not accepting my changes. So I don't forget for later, I'm opening this as an issue. (Which itself is an indication the problem is resolved, but whatever)

Gluttony actually increases foodvalues by 50% per level. So, level 3 is a 150% increase, not a 200% increase.

**Comment by James103** [comment]
The wiki page for Fasting should include:
"If a chestplate is cheated to have a higher level of Fasting than the expected maximum, the pattern continues. At level 10 or higher, the player receives zero exhaustion."

Additionally, on the wiki page for Fasting, "this enchantment reduces a player's exhaustion by 10% per level." should be "this enchantment reduces a player's exhaustion **rate** by 10% per level."

The wiki page for Gluttony should include:
"If a chestplate is cheated to have a higher level of Gluttony than the expected maximum, each additional level increases the food values by 50%."

The wiki page for Metabolism (and for Spice of Life: Carrot Edition, Commands, and Config) appears to result in an access denied error, with "You do not have permission to update this wiki". Most likely this means the page either does not exist or has restricted read access privileges.

**Comment by yeelp** [comment]
Yes, some pages for the wiki I haven't gotten around to making yet, which I really ought to do.

These other changes are good though.

**Comment by James103** [comment]
The wiki page for Famine (and other locations where applicable) should state that the effect durations are in seconds. Currently, it's hard to tell whether the duration is in ticks or in seconds.

**Comment by yeelp** [comment]
I'm going to save the Famine page for after 1.4, since it's receiving major changes.

Metabolism now has a wiki page
Gluttony now has a much better page
missing icons not related to saturation now exist.
The corrections you mentioned have otherwise been made.
The SoL:Carrot Edition, Commands and Config pages still don't exist yet. The config page in particular will be long and I'm not sure how I'm going to structure it yet.

**Comment by James103** [comment]
Here are the rest of the values for the Metabolism potion effect's causes:
**Source**|**Potency**|**Duration**
:-----:|:-----:|:-----:
Splash Metabolic Potion (Extended)|I|4:00
Splash Metabolic Potion II|II|1:00
Lingering Metabolic Potion|I|0:30
Lingering Metabolic Potion (Extended)|I|1:00
Lingering Metabolic Potion II|II|0:15
Metabolic Arrow|I|0:15
Metabolic Arrow (Extended)|I|0:30
Metabolic Arrow II|II|0:07*

*One-half of the 15 seconds of Metabolic Arrow I, displayed rounded down. Exact value without rounding: 150 ticks (7.5 seconds).

**Comment by yeelp** [comment]
Excellent, I hadn't gotten around to hopping in game to check (I just kept forgetting lol). I'll update the wiki right now.

**Comment by James103** [comment]
Here are some corrections that I would like to make to the Famine page. They update the page to the latest mod version (1.4).

The first paragraph should have any mentions of the Hunger effect removed and replaced with the fact that players get an instant deduction to their food level and that this penalty scales by the enchantment level.

The corrected table is as follows:
**Enchantment Level**|**Hunger Loss**|**Weakness Level**
:-----:|:-----:|:-----:
1|1.25|1
2|2.5|1
3|3.75|2
4|5|2
5|6.25|2

The second paragraph should also have any mentions of the Hunger effect removed. "the Hunger effect duration scales by level" should be replaced with "the amount of food points lost scales by level, up to a maximum of 10 food points per hit.".

Additionally, the fact that a full charge is required to activate the Famine enchantment should be mentioned there as well.


Outside of the Famine wiki page, there are some more errors that need to be corrected:
* "That is, with Famine III and Iron Stomach II" should be "That is, with **Fasting** III and Iron Stomach II" (on the Fasting wiki page)
* "For example, a level 5 Eternal Feast will award the player 10 () hunger" should be "For example, a level 5 Eternal Feast will award the player 10 () hunger **per kill**" (on the Eternal Feast wiki page)

**Comment by yeelp** [comment]
Done! Thanks for pointing this out!

**Comment by James103** [comment]
On the Metabolism wiki page, where it says "Tick per [food point]" should say "Tick**s** per [food point]".

**Comment by James103** [comment]
The following Wiki pages have not been created yet, but the corresponding features are still in the mod:
#### Enchantments
* Curse of Laziness
* Curse of Deprivation
* Curse of Sensitivity
#### Potion effects
* Bloated
* Deficiency
* Hunger Plus
* Hunger Minus
* Soft Stomach
#### Attributes
* Max Hunger
* Food Efficiency
#### Other
* Config (page exists but no actual content)
* GUI (page exists, but uncategorized and no actual content)

Additionally, the page "spiceoflifemodule" should be moved (renamed) to "Spice of Life Module" or "The Spice of Life (Module)".

**Comment by yeelp** [comment]
Enchantments are written now, I'll get to the potion effects in the next couple of days.

The GUI page actually shouldn't be there. Instead, whatever content would've been there should be on the HUD page instead (which is more technically correct. Scaling Feast doesn't add any GUI's of its own, it just alters the HUD).

**Comment by James103** [comment]
> Additionally, the page "spiceoflifemodule" should be moved (renamed) to "Spice of Life Module" or "The Spice of Life (Module)".

This has broken the corresponding link on your custom wiki navigation sidebar. The internal link still points to the old name (spiceoflifemodule) instead of the new name (Spice-of-Life-Module).

**Comment by yeelp** [comment]
Right, should've checked that. Thanks.

**Comment by James103** [comment]
Are you still working on the potion effect pages? It's been a week since your last contribution, and the potion effect/attribute pages still have not been written.

**Comment by yeelp** [comment]
Yes I'm still working on them. I've been busy finishing some summer exams the past couple of days. I've been writing the general text offline which I'll stylize with Markdown when I update the wiki pages.

**Comment by James103** [comment]
> The config page in particular will be long and I'm not sure how I'm going to structure it yet.

I think there first should be a table of contents to jump to a particular config option (or a config section thereof), and then the config options should be placed in the same order and directory tree as they appear in-game (or on the config files). Each config option should have at the very least all of the same info as it appear in game (so name, accepted values, default value, what it does, etc).

**Comment by yeelp** [comment]
This is a good idea. I was thinking about also putting some suggested configurations in there too, for those curious about the kind of gameplay they can get from Scaling Feast. Putting said list together may be time consuming, so I may omit it.
